lua/trim/init.lua

```diff
@@ -73,4 +74,10 @@ function M.disable()
   vim.notify('TrimNvim disabled', vim.log.levels.INFO, { title = 'trim.nvim' })
 end
 
+function M.trim()
+  if not has_value(M.config.blacklist, vim.bo.filetype) then
```

Comment:
> Thanks for your PR.
> 
> In this PR, let's avoid breaking compatibility. Please make no changes from `diable` to `blacklist`.

As `disable` is more like a boolean option to disable the plugin totally, I think it's better to move to `blacklist`.

For compatibility issue, [this commit](url)  add another check to handle the case, it will try to use `disable` if `blacklist` is not present, the logic is simple and should be enough for seamless switch 🙂